core/kazoo_services/src/kz_services.erl

```diff
@@ -429,14 +475,18 @@ ratedeck_id(Services) ->
 
 -spec plan_ratedeck_id(services()) -> kz_term:api_ne_binary().
 plan_ratedeck_id(Services) ->
-    Plans = kz_services_plans:foldl(fun(_BookkeeperHash, PlansList, Plans) ->
-                                            PlansList ++ Plans
-                                    end
-                                   ,[]
-                                   ,plans(Services)
-                                   ),
-    Plan = kz_services_plans:merge(Plans),
-    kz_services_plan:ratedeck_id(Plan).
+    case kz_services_plans:foldl(fun(_BookkeeperHash, PlansList, Plans) ->
```

Comment:
We've been trying to move folds like this into their own nicely-named function to make it easier to know what's being done in the fold. For instance, maybe `case collect_plans(plans(Services) of`. Also, using a named fun for the fold is preferred if no state is being used from outside the fun()'s scope.